modules/charging_station/route_algorithm.py

```diff
@@ -0,0 +1,92 @@
+'''
+Thoughts:
+valid path from start to finish
+1. segment where total distance is less than C between recharge. Only move along path where distance within two nodes is less than C meters
+2. stop at charging stations when needed
+3. lowest possible distance from start to finish (regardless of no of stops/refilling)
+
+** keep track of remaining distance at every point in time?
```

Comment:
This is a good insight: how would we keep track of the remaining distance? Where would this information go?